### PR TITLE
Fix security vulnerability in cypress/support/utils.ts

### PR DESCRIPTION
**Pull request by hayescode**
Fixes issue #983

### CODE REVIEW

File: cypress/support/utils.ts [hunk 1]
```diff
@@ -24,10 +24,13 @@ export async function runTests(matchName) {
 }
 
 export function runCommand(command: string, cwd = ROOT) {
-  return execSync(command, {
+  const [cmd, ...args] = command.split(' ');
+  const fullCommand = [cmd, ...args].join(' ');
```
Comment (willydouhard) [inline cypress/support/utils.ts:28]: in the end the command being ran is the same right? Your security scanner happens to prefer this syntax?

File: cypress/support/utils.ts [hunk 2]
```diff
@@ -24,10 +24,13 @@ export async function runTests(matchName) {
...
+  return execSync(fullCommand, {
     encoding: 'utf-8',
     cwd,
     env: process.env,
-    stdio: 'inherit'
+    stdio: 'inherit',
+    shell: process.platform === 'win32' ? 'cmd.exe' : '/bin/sh',
```
Comment (willydouhard) [inline cypress/support/utils.ts:34]: can you explain this addition?

File: cypress/support/utils.ts (hunk 1 above)
Comment (hayescode) [reply cypress/support/utils.ts:28]: @willydouhard i based this on gpt4 feedback but you're right, this doesn't seem to remove the vunerability. The scanner doesn't run every day and the last check (where I manually added this) still has it.

Any other ideas?

**Comment by dokterbob** [comment]
Closing for now. Rationale: https://github.com/Chainlit/chainlit/issues/983#issuecomment-2285848329

**Pull request closed without merging**